### PR TITLE
gcc: fix AVX512

### PR DESCRIPTION
**Pull request by Andarwinux**
Fixed crash with -DGCC_ARCH=tigerlake/znver4.

### CODE REVIEW

File: toolchain/gcc/gcc.cmake [hunk 1]
```diff
@@ -37,6 +37,7 @@ ExternalProject_Add_Step(gcc final
         rustup
     COMMAND ${MAKE}
     COMMAND ${MAKE} install-strip
+    COMMAND bash -c "rm -rf ${MINGW_INSTALL_PREFIX}/bin/{ld,ld.bfd}"
```
Comment (shinchiro) [inline toolchain/gcc/gcc.cmake]: This not needed..compiler always start with cross-* prefix

Comment (Andarwinux) [reply toolchain/gcc/gcc.cmake]: I found out via htop that it actually use this ld.

Comment (shinchiro) [reply toolchain/gcc/gcc.cmake]: There's no ld or ld.bfd file in that bin folder, so that rm command is just pointless

Comment (Andarwinux) [reply toolchain/gcc/gcc.cmake]: My local build does have this ld here. but can delete this, currently the bfd wrapper is unused.

Comment (shinchiro) [reply toolchain/gcc/gcc.cmake]: > currently the bfd wrapper is unused

How you check it?

Comment (Andarwinux) [reply toolchain/gcc/gcc.cmake]: `-DBFD_FLAGS="--major-os-version=10"`
`llvm-readobj --headers mpv.exe`

**Pull request merged**